src/test/java/seleniumTasks/SixPMTest.java

```diff
@@ -0,0 +1,84 @@
+package seleniumTasks;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.*;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.Random;
+
+public class SixPMTest {
+    private WebDriver driver;
+    private WebDriverWait wait;
+    private Actions action;
+
+    @BeforeSuite
+    public void setUps() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://www.6pm.com");
+        action = new Actions(driver);
+        wait = new WebDriverWait(driver, 10);
+    }
+
+    @Test
+    public void addToBag() {
+        By accessoriesInMenuBarLoc = By.xpath("//a[@href='/c/accessories']");
+        wait.until(ExpectedConditions.visibilityOfElementLocated(accessoriesInMenuBarLoc));
+        WebElement accessoriesInMenuBar = driver.findElement(accessoriesInMenuBarLoc);
+        action.moveToElement(accessoriesInMenuBar).perform();
+
+        By aviators = By.xpath("//a[contains(text(), 'Aviators')]");
+        wait.until(ExpectedConditions.visibilityOfElementLocated(aviators));
+        try {
```

Comment:
What the reason of this `try-catch` block here? Why do you think the element will not be found? You have an explicit wait just 1 line above for visibility of this element.